auacm/app/modules/competition_manager/views.py

```diff
@@ -158,10 +158,16 @@ def get_competition_data(cid):
     scoreboard = list()
 
     team_users = dict()
+    team_display_names = dict()
+
     for user in comp_users:
         if not user.team in team_users:
             team_users[user.team] = list()
+            team_display_names[user.team] = list()
         team_users[user.team].append(user.username)
+        team_display_names[user.team].append(database.session.query(User)
+                                            .filter(User.username == user.username)
+                                            .first().display)
```

Comment:
This creates a new query for every user in a competition. I don't want to prematurely optimize, but I would be interested if it would be faster to get all the display names (for users in the competition) in one query and stick it in a dictionary for lookup. This may be moot if SQLAlchemy does any kind of caching. I'll do a quick performance test and see if there's a difference.